crates/uv-trampoline/src/bounce.rs

```diff
@@ -509,6 +499,12 @@ pub fn bounce(is_gui: bool) -> ! {
     exit_with_status(exit_code);
 }
 
+#[cold]
+fn error_and_exit(message: &str) -> ! {
```

Comment:
nit: would prefer like `exit_with_error`?

Reply:
I guess probably not worth the regen

Reply:
(unless needed to test https://github.com/astral-sh/uv/pull/11295#discussion_r1946976564)